python/cudf/cudf/_lib/sort.pyx

```diff
@@ -208,13 +233,70 @@ def sort(
     return columns_from_unique_ptr(move(result))
 
 
+@acquire_spill_lock()
+def sort_by_key(
+    list values,
+    list keys,
+    object ascending,
+    object na_position,
+    *,
+    bool stable,
+):
+    """
+    Sort a table by given keys
+
+    Parameters
+    ----------
+    values : list[Column]
+        Columns of the table which will be sorted
+    keys : list[Column]
+        Columns making up the sort key
+    ascending : list[bool]
+        Sequence of boolean values which correspond to each column in
+        keys providing the sort order (default all True).
+        With True <=> ascending; False <=> descending.
```

Comment:
Maybe match the docstring from `order_by`? This one has very slightly different text.

Reply:
Done, thanks